### PR TITLE
MRCD8-218 added visual indicators for meatballs and arrows

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Added visual indicators for focus on meatballs below slideshow and to arrows that overlay images

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the focus state for the meatballs and the previous and next buttons now have focus states (leveraging the Decanter $focus-shadow variable)

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-218

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by pookmish — CHANGES_REQUESTED** [review]
I think it needs a much stronger effect than a box shadow for visually impaired users. Even i had a hard time telling where my focus was

**Comment by pookmish** [comment]
And in fact we need complete focus styling for the entire site. I bet if you do some focus style for the genera links and buttons, it'll apply to the slideshow as well

File: themes/math_research_center/scss/paragraph/mrc_slideshow.scss [hunk 1]
```diff
@@ -78,6 +78,10 @@
 
         @include padding(0 10px);
       }
+
+      &:focus:before {
+        box-shadow: $focus-shadow;
```
Comment (pookmish) [inline themes/math_research_center/scss/paragraph/mrc_slideshow.scss]: So do we need these now that we have the mixin?

**Comment by josephgknox** [comment]
@pookmish nope -- they're gone! :)

**Comment by pookmish** [comment]
Still getting ```a:focus {
    outline: 0;
}```
that overrides that mixin.

**Pull request merged**